ansible/roles/baselayout/tasks/main.yml

```diff
@@ -82,6 +82,17 @@
     - "{{ packages[os|stripversion]|default('[]') }}"
     - "{{ common_packages|default('[]') }}"
 
+- name: install packages IBMi
+  when: os|startswith("ibmi")
+  yum: name="{{ package }}" state=present
+  loop_control:
+    loop_var: package
+  with_items:
+    # ansible doesn't like empty lists
+    - "{{ packages[os]|default('[]') }}"
+    - "{{ packages[os|stripversion]|default('[]') }}"
+    - "{{ common_packages|default('[]') }}"
```

Comment:
This task is identical to the previous one, isn't it? The one where you added an exclude for ibmi.

Reply:
Rather than `package:`, the ibmi leg has `yum:`.

Reply:
Ah, so it does. Okay, comment withdrawn.